rcfc/static/index.js

```diff
@@ -29,17 +29,58 @@
         }
     }
 
+    function displayGroup(group) {
+      if(group == null) {
+        name = "unassigned"
+      } else {
+        name = group
+      }
+      let id = name.replace(/ /g, '_');
```

Comment:
this was my fix for the webserver.. Introduces a bug where 'Group A' and 'Group_A' are seen as the same thing.

Reply:
I should be more clear with my variables. This isn't the id of the 'button' from the context of the user, but rather the id of the html button for specifying the group. When reading, it seemed like html didn't like that, but after doing further digging I might be able to get away with it in this context: https://stackoverflow.com/questions/6802765/jquery-dealing-with-a-space-in-the-id-attribute

Reply:
I'm not extremely familiar with building UIs with html/JS, so I thought I needed the id of the button to assign the function to the button.